examples/master/master.ino

```diff
@@ -2,20 +2,124 @@
 
 bool addresses[127] = {false};
 
-// Addresses pins
-int as[] = {8, 9, 10, 11};
+uint16_t timeleft = 300;   // 5 minutes
+uint16_t difficulty = 1;   // Easy
 
-uint16_t timeleft = 300; // 5 minutes
-uint16_t difficulty = 1;
+bool defused = false;      // Is the bombe defused ?
 
-void initAddresses()
+// Message's structure
+struct BusMessage
 {
-  for (int i = 0; i < sizeof(as) / sizeof(as[0]); i++) {
-    pinMode(as[i], OUTPUT);
-    digitalWrite(as[i], HIGH);
+  uint16_t data[2];
+};
+
+// Output
+#define RED_LED (2)
+#define GREEN_LED (3)
+#define YELLOW_LED (4)
+
+// Input
+#define SENSE_PIN (0)
+
+// Command
+# define TIME_CMD (0)
+# define DIFFICULTY_CMD (1)
+# define NEED_TO_SPEAK_CMD (2)
+# define INFO_CMD (3)
+
+# define DEFUSED_CMD (4)
+# define PENALITY_CMD (5)
+# define END_CMD (6)
+
+// Answer
+# define ANSWER_YES (1)
+# define ANSWER_NO (0)
+# define ANSWER_OK (1)
+
+void printDeviceFound(uint8_t address)
```

Comment:
Somewhere, device addresses are treated as `int`s (16 bytes, potentially 8), and sometimes they are clearly 8 bytes int. grand unification plan needed